front/app/modules/commercial/customizable_homepage_banner/index.tsx

```diff
@@ -10,14 +10,15 @@ import CTA from './citizen/CTA';
 const configuration: ModuleConfiguration = {
   outlets: {
     'app.containers.Admin.settings.customize.headerSectionStart': (props) => (
-      <FeatureFlag name="customizable_homepage_banner">
-        <LayoutSetting {...props} />
-      </FeatureFlag>
+      // these are no longer in feature flags but in the homepage config
```

Comment:
placeholder for now, until we figure out how to manage when allowed/enabled are coming from two places